src/foam/flow/DocumentationFolderDAO.js

```diff
@@ -35,28 +35,26 @@ sink = prepareSink(sink);
 foam.dao.Sink         decorated = decorateSink_(sink, skip, limit, order, predicate);
 foam.dao.Subscription sub       = new foam.dao.Subscription();
 
-java.nio.file.FileSystem fs = java.nio.file.FileSystems.getDefault();
-java.nio.file.DirectoryStream<java.nio.file.Path> contents;
+// Load files from both the Jar and regular FileSystem
+java.util.Map<String, java.io.InputStream> iRStreamMap = new foam.nanos.fs.Storage(getDir(), true).getDirectoryAsStream("");
```

Comment:
Do we really need fallback support?  I don't think we'll ever actually be serving from both places at the same time, we either serve from the filesystem  Rather than redo most of the code of this class we only need to be smarter about which FileSystem we start from.

We can crib from how HttpServer does it here: https://github.com/foam-framework/foam2/blob/master/src/foam/nanos/jetty/HttpServer.js#L114

Basically if nanos.webroot is set, then we use that as the resource base.  We can do something like

```
String documentFs = System.getProperty("nanos.document.fs");
if ( documentFs == null ) {
  documentFs = System.getProperty("user.dir");
}
java.nio.file.FileSystem fs = java.nio.file.FileSystem.getFileSystem(new URI(documentFs));
```

and then set nanos.document.fs appropriately when running from a JAR or not.

and then leave the rest as is.  The above code might need some tweaking.

Reply:
@adamvy But what about edited flow documents - flow documents shipped in the jar will then have a later version outside the jar.

Reply:
@jlhughes that's what we should have, we shouldn't be updating the JAR IMO, the jar we ship should be immutable.